master/static/migrations/20211221113503_add-allocation-state.down.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE public.allocations
+    DROP COLUMN state;
+
+ALTER TABLE public.allocations
+    DROP COLUMN isReady;
```

Comment:
if we go this route, i think `is_ready` is in line with all the rest of the columns in the db.

Reply:
bump